services/user-tenant-service/src/main/java/com/sourcefuse/jarc/services/usertenantservice/controller/GroupController.java

```diff
@@ -0,0 +1,95 @@
+package com.sourcefuse.jarc.services.usertenantservice.controller;
+
+import com.sourcefuse.jarc.services.usertenantservice.DTO.Count;
+import com.sourcefuse.jarc.services.usertenantservice.DTO.Group;
+import com.sourcefuse.jarc.services.usertenantservice.DTO.UserGroup;
+import com.sourcefuse.jarc.services.usertenantservice.exceptions.ApiPayLoadException;
+import com.sourcefuse.jarc.services.usertenantservice.exceptions.GenericRespBuilder;
+import com.sourcefuse.jarc.services.usertenantservice.service.GroupService;
+import com.sourcefuse.jarc.services.usertenantservice.service.UserGroupsService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@Slf4j
+@RequestMapping(value = {"${api.groups.context.url}"})
```

Comment:
url endpoint cannot be picked from envs please fix this

Reply:
done